e2e/backend_service_update_test.go

```diff
@@ -31,12 +28,19 @@ func buildBackendServiceWithLBScheme(graphBuilder *rgraph.Builder, name string,
 		x.TimeoutSec = 30
 		x.ConnectionDraining = &compute.ConnectionDraining{}
 		x.HealthChecks = []string{hcID.SelfLink(meta.VersionGA)}
-	})
+	}
+
+	return buildBackendServiceWith(graphBuilder, name, f)
+}
+
+func buildBackendServiceWith(graphBuilder *rgraph.Builder, name string, f func(x *compute.BackendService)) (*cloud.ResourceID, error) {
```

Comment:
`buildBackendServiceWith` -> `buildBackendService`

Reply:
there is also `buildBackendServiceWithNEG` please update this function as well

Reply:
there already exists a function buildBackendService, not renaming.

Reply:
Done.